sema/check/sinks/html_sink.py

```
@@ -0,0 +1,19 @@
+# your_submodule/sinks/html_sink.py
+
+from typing import List
+from ..testing.base import TestResult
+
+
+def write_html(results: List[TestResult], output_file: str):
```

Ruff

Missing return type annotation for public function `write_html`

Add return type annotation: `None`

(ANN201)